src/core/template-wrapper.ts

```diff
@@ -18,6 +19,14 @@ class TemplateWrapper extends React.PureComponent<ITemplateWrapperProps, object>
     }
 
     public componentDidMount() {
+        if (this.props.onRendered) {
+            setTimeout(() => {
+                if (this.props.onRendered) {
```

Comment:
Is this double-check required? I believe one can be omitted

Reply:
The second check is related with TS compilation